### ISSUE TITLE
Google Drive new Doc bug

### ISSUE THREAD

**Issue by Zaraklin** [brave/browser-laptop#2070]
Opened drive.google.com
Click new > Document
App totally crashed.

Platform: Windows 10
Brave Version: 0.10.1

**Comment by cezaraugusto** [comment]
**Testing on Brave versions:**

* 0.10.0 (OSX 10.11.4, Windows 8.1, Windows 10)
* 0.10.1dev (OSX 10.11.4, Ubuntu, Windows)
* 0.10.1 (OSX 10.11.4, Windows 8.1, Windows 10)
* 0.10.2 (Windows 8.1, Windows 10)

**Results:**

* Can't reproduce the error on OSX 10.11.4
* Can't reproduce the error on Linux Ubuntu 14.04
* Can't reproduce the error on Windows 8.1, Windows 10 (both x64)

Could you describe more fully how the app crashed? Also:

* Did you build Brave yourself or has you downloaded the executable from http://brave.com?
* If you build it yourself, which Node version (`node -v`) are you running?
* Had you tried restarting your system and reproducing the error again? Does it persist?

**Comment by bbondy** [comment]
Thanks, this is fixed with 0.10.2.

**Comment by Zaraklin** [comment]
How it crashed:
I opened google drive site, logged in my account and then clicked in the option to create a new Document (Google Docs). After that, the app crashed. In the first time, windows show me a message saying that the app stoped working, in the others tries the app just close.

I installed with the executable when it was in the 0.9.4 version, and keep using it and updating when the message apears.
Yes, the problem persists after restarting.
